core/README.md

```diff
@@ -43,10 +43,11 @@ Other properties are also allowed to be provided, but are not described by this
 
 ### General Properties
 
-| Property Name | Data Type | Description |
-| ------------- | --------- | ----------- |
-| id            | string    | **REQUIRED.** A unique identifier for the field. It must be unique within the [Collection](#collection). |
-| collection    | string    | The identifier of the parent collection. |
+| Property Name  | Data Type | Description |
+| -------------- | --------- | ----------- |
+| id             | string    | **REQUIRED.** A unique identifier for the field. It must be unique within the [Collection](#collection). |
+| collection     | string    | The identifier of the parent collection. |
+| administrative | boolean   | Determines whether the boundary is coming from an official (usually governmental) source. |
```

Comment:
We may want to think about a more generic category field, instead of a specific administrative boolean flag.

For example: => category with enum: conceptual, economic, operational, administrative, other

Source: https://aggateway.org/Portals/1010/WebSite/About%20Us/FIELD%20BOUNDARY%20FLYER%20122123.pdf?ver=2024-01-03-212959-590